src/murmur/Server.cpp

```diff
@@ -465,9 +430,7 @@ void Server::readParams() {
 	bCertRequired      = getConf("certrequired", bCertRequired).toBool();
 	bForceExternalAuth = getConf("forceExternalAuth", bForceExternalAuth).toBool();
 
-	qvSuggestVersion = normalizeSuggestVersion(getConf("suggestversion", qvSuggestVersion));
-	if (qvSuggestVersion.toUInt() == 0)
-		qvSuggestVersion = QVariant();
+	m_suggestVersion = Version::getRaw(getConf("suggestversion", m_suggestVersion));
```

Comment:
In contrast to the old implementation, this new implementation seems to only allow bitmasked suggestversion values. The old implementation also allowed for version strings. We should retain that feature.

Reply:
That is pretty funny because the new implementation ONLY allows strings as ``suggestversion``. Internally it is held in a ``mumble_version_raw_t``, yes. But the settings are supposed to only contain strings. The funny part is that I am not the only one confusing ``getRaw`` with ``toRaw``;)

Reply:
Is it ok, if new server only accept strings as suggested version settings? Old bit masked integers will be broken anyway when the server is upgraded and these will not be future proof.

Reply:
Since we intend to backport these changes, we can't drop any functionality here.
And old bitmasked values should simply be imported via the legacy functions as is done everywhere else. We can easily check whether a bitmask is in the old or new format simply by checking its numeric value. Since the minor and patch version in the new format take more bytes, masks in the new format will necessarily be much bigger than in the old format (assuming that every version has at least 1 as the major version component, which is a sane assumption)

Reply:
@Krzmbrzl I was thinking really hard for some time why I felt like this did not make sense to add here, but now I finally figured that I changed the type of ``m_suggestversion`` to ``version_raw_t`` in ``Meta.cpp``. I am divided whether or not to store the suggest version as raw int or string in the interal config, but I am leaning towards string, because it does not depend on the format used.
That means that i need to accept masked ints in Meta.cpp instead of Server.cpp.
I think... We will see...

Reply:
Supporting bit masked ints as setting AND allowing these bit masked ints to be of the old format will produce very ugly code